pvlib/shading.py

```diff
@@ -232,3 +232,84 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def projected_solar_zenith_angle(axis_tilt, axis_azimuth,
+                                 solar_zenith, solar_azimuth):
+    r"""
+    Calculate projected solar zenith angle in degrees.
+    This is the solar zenith angle projected onto the tracker rotation plane or
+    the plane defined by its normal vector and the azimuth.
```

Comment:
```suggestion
    This solar zenith angle is projected onto the plane whose normal vector is defined by
    ``axis_tilt`` and ``axis_azimuth``.  The normal vector is in the direction of ``axis_azimuth``
    (clockwise from north) and tilted from horizontal by ``axis_tilt``. See Figure 5 in [1]_.
```

One picture would be worth many words here.

Reply:
Thanks for upgrading the definition.

Regarding copyright, is it possible to add that Figure 5 here (if you consider it a good addition)? Or is it better to do something new by myself? I had previously started doing a diagram:

![image](https://github.com/pvlib/pvlib-python/assets/80125792/335a3040-a60a-4b18-976b-9b325ce15778)

It still lacks some vectors and angles, and IMO has an excess of details, anyway. Just in case it is a starting point.

Reply:
We can add that Figure here, with a citation. I think your version with the colored planes would be easier to read but: 1) theta should be an angle in the purple plane, it looks different here 2) indicate the plane's normal vector 3) indicate the solar zenith angle 4) show the tilt and azimuth relative to a N-E-S-W coordinate system 5) label the sun, and 6) add direction to the angles so the sign convention is clear.

Reply:
Yes, I see my figure is completely wrong for this purpose. I've got some issues regarding the comprehension of this PSZ concept. If you are ok with it, I'll be adding said Figure 5.

Reply:
Image added!

https://pvlib-python--1904.org.readthedocs.build/en/1904/reference/generated/pvlib.shading.projected_solar_zenith_angle.html